app/src/main/java/com/globant/harrypotterapp/activity/MainActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.globant.harrypotterapp.activity
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.globant.harrypotterapp.R
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+    }
+    // TODO: implement SpellActivity
+
+    companion object {
+        fun getIntent(activity: Activity): Intent {
```

Comment:
you can use this here:
fun getIntent(activity: Activity): Intent = Intent(activity, MainActivity::class.java)

Reply:
Yes, you are right, thanks Mau! Refactored